lib/ansible/modules/net_tools/nmcli.py

```diff
@@ -702,7 +702,7 @@ def up_connection(self):
         cmd.append('con')
         cmd.append('up')
         cmd.append(self.conn_name)
-        return self.execute_command(cmd)
+        return cmd
```

Comment:
this change breaks the pattern of other methods like down_connection (1/2 of the methods are constructors the other 1/2 executors) so i would change the caller instead.